src/lti_provider/settings.py

```diff
@@ -0,0 +1,38 @@
+"""This module contains default settings for lti_provider application."""
+
+from django.conf import settings
+
+# When a LTI launch request is successfully verified, the handler function defined
+# in the LTI_LAUNCH_SUCCESS_HANDLER setting is called with 2 parameters :
+# 1. The request (django.http.request.HttpRequest)
+# 2. The verified LTI request (lti_provider.lti.LTI)
+#
+# The handler should return a valid HTTP response (django.http.HttpResponse)
+# A default implementation is given in this application as an exemple
+#
+# If you use the default implementation, you must add the following
+# authentication backend in your AUTHENTICATION_BACKENDS setting:
+#   "lti_provider.default.backend.AuthBackend"
+#
+
+settings.LTI_LAUNCH_SUCCESS_HANDLER = getattr(
+    settings, "LTI_LAUNCH_SUCCESS_HANDLER", "lti_provider.default.handlers.success"
+)
+
+# When a LTI launch request is invalid, the handler function defined in the
+# LTI_LAUNCH_FAILURE_HANDLER is called with the following parameters:
+# 1. The request (django.http.request.HttpRequest)
+# 2. The LTIException
+#
+# The handler should return a valid HTTP response (django.http.HttpResponse)
+# A default implementation is given in this application as an example.
+settings.LTI_LAUNCH_FAILURE_HANDLER = getattr(
+    settings, "LTI_LAUNCH_FAILURE_HANDLER", "lti_provider.default.handlers.failure"
+)
+
+# Name of the Django cache to use for the replay attack protection.
+# Timestamp and nonce used to sign LTI request will be stored in this cache
+# to prevent replay attacks.
+settings.LTI_REPLAY_PROTECTION_CACHE = getattr(
+    settings, "LTI_REPLAY_PROTECTION_CACHE", "default"
```

Comment:
Then you can just use a cache called "lti_replay" for example and the Django cache framework automatically falls back to default if this cache is not declared in the settings.

Reply:
good to know, thanks !

Reply:
Hmm it doesn't seem to work when I ask for a cache that is not declared.
I've got an `InvalidCacheBackendError` (_Could not find config for 'lti_replay' in settings.CACHES_) thrown by Django's CacheHandler :
https://github.com/django/django/blob/stable/2.2.x/django/core/cache/__init__.py#L74-L77

Reply:
oups sorry. So make a try except at the place where you get the cache?

Reply:
:heavy_check_mark: